### PR TITLE
Reorganize and prune functions

### PR DESCRIPTION
**Pull request by chendaniely**
Cleaning up the codebase

### CODE REVIEW

**Comment by chendaniely** [comment]
@schloerke I haven't gotten to fixing `test_result` yet, but want to make sure if the `test` and `tests` functions are used.

**Comment by schloerke** [comment]
am I still to review this one?

**Comment by chendaniely** [comment]
yep. just after this travis test.

**Comment by chendaniely** [comment]
@schloerke all ready for review.

File: R/test_solutions.R [hunk 1]
```diff
@@ -1,162 +0,0 @@
...
-#'   is listed by name alongside a result status. Within the RStudio IDE, chunks
-#'   that run without an error or warning are printed with a green checkmark.
-#'   Chunks that run but return a warning message are printed with a yellow
-#'   checkmark and the warning message. Chunks that produce an error are printed
-#'   with a red x and the error message. Chunks that contain an error related to
-#'   how learnr processes the chunk contain are printed with a purple error
-#'   symbol followed by a message.
-#'
-#' @export
-test_solutions <- function(file = NULL,
```
Comment (schloerke) [inline R/test_solutions.R]: want to put this into `./archive`?

File: archive/result.R [hunk 2]
```diff
@@ -13,13 +13,13 @@
 #' result(3, "Custom message for value 3.")
 #' result(4, "Custom message for value 4.", correct = TRUE)
 #' \dontrun{grading_demo()}
-result <- function(x, message = NULL, correct = FALSE) {
-  chkm8_single_character(message)
-  checkmate::expect_logical(correct, any.missing = FALSE, len = 1, null.ok = FALSE)
+# result <- function(x, message = NULL, correct = FALSE) {
```
Comment (schloerke) [inline archive/result.R]: delete this one (docs too). I made it. Not being used anymore.

File: archive/test_result.R [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#' TODO document with 'tests' documentation
+#' @export
+#' @rdname test
+#' @param message Message to report back if the test throws an error.
+#' @param fn function to execute against the user solution.
+#'   If the test fails, it should throw an error to display the \code{message} provided.
+# test <- function(message, fn) {
```
Comment (schloerke) [inline archive/test_result.R]: delete this one. I made it. It is not needed anymore

File: R/test_solutions.R (hunk 1 above)
Comment (chendaniely) [reply R/test_solutions.R]: already moved to `archive`

File: archive/result.R (hunk 2 above)
Comment (chendaniely) [reply archive/result.R]: docs already deleted.

File: archive/test_result.R (hunk 3 above)
Comment (chendaniely) [reply archive/test_result.R]: docs already deleted

**Pull request merged**